### PR TITLE
[#17357] move status-im.async-storage.core

### LINKED ISSUE #17357: move status-im.async-storage.core

### PR DESCRIPTION
**Pull request by flexsurfer**
fixes #17357

### CODE REVIEW

File: src/status_im/communities/core.cljs [hunk 1]
```diff
@@ -819,13 +818,6 @@
...
 
-(rf/defn store-category-state
-  {:events [::store-category-state]}
-  [{:keys [db]} community-id category-id state-open?]
-  (let [public-key (get-in db [:profile/profile :public-key])
-        hash       (category-hash public-key community-id category-id)]
-    {::async-storage/set! {hash state-open?}}))
-
 (rf/defn update-category-states-in-db
   {:events [::category-states-loaded]}
```
Comment (Parveshdhull) [inline src/status_im/communities/core.cljs]: this is also outdated

**Review by Parveshdhull — APPROVED** [review]

**Comment by flexsurfer** [comment]
@status-im/mobile-qa could someone pls take a look if e2e is ok to merge?

**Comment by VolodLytvynenko** [comment]
> @status-im/mobile-qa could someone pls take a look if e2e is ok to merge?

Hi @flexsurfer I am checking e2e. Let you know soon if something is introduced by this PR

**Comment by VolodLytvynenko** [comment]
Hi @flexsurfer here is the issue found in e2e:

## ISSUE 1: When the app is killed and then reopened, the chat does not open as expected.

### Steps to Reproduce:
1. Open any chat.
2. Kill the app (force close it).
3. Reopen the app.

### Actual Result:
The chat that was previously open does not automatically reopen after killing and reopening the app.

https://github.com/status-im/status-mobile/assets/52490791/536311b2-0aa3-49bd-8083-f2eca4734cb9


### Expected Result:
When the app is reopened, it should restore the state of the previously opened chat

**Comment by VolodLytvynenko** [comment]
@flexsurfer Other e2e issues are not related to this PR

**Comment by flexsurfer** [comment]
thanks @VolodLytvynenko fixed

**Comment by VolodLytvynenko** [comment]
@flexsurfer Today there are problems with message delivery (specifically in e2e tests). Which leads to e2e constant failures. I'll rerun  e2e again. If the picture is still unclear, then I’ll take this PR and retest it manually.
Probably any known areas that could be affected by this PR?

**Comment by flexsurfer** [comment]
> Probably any known areas that could be affected by this PR?

i don't think there are much, if chat works fine thats should be ok, thanks

**Comment by VolodLytvynenko** [comment]
> i don't think there are much, if chat works fine thats should be ok, thanks

Thanks for the clarifications. No issues for Android builds. But I am unable to check it on the ios. IOS was not built successfully

**Comment by flexsurfer** [comment]
@VolodLytvynenko rebased

**Comment by VolodLytvynenko** [comment]
> @VolodLytvynenko rebased

@flexsurfer Thank you for PR. No issues from my side. Can be merged

**Pull request merged**